src/pages/main/__test__/Editor.test.tsx

```diff
@@ -25,10 +31,22 @@ describe('<Editor/>', () => {
     jest.clearAllMocks();
   });
 
-  it('should render with all children', async () => {
-    const { getByTestId } = await act(() => renderEditor());
-    getByTestId('latex-text-area');
+  it('should render with all children', () => {
+    const { getByTestId } = renderEditor();
```

Comment:
:-1:

Reply:
I was testing something and left it, added it back